engine/gui/src/dmsdk/gui/gui.h

```diff
@@ -117,7 +117,7 @@ namespace dmGui
      * @member NODE_TEXTURE_TYPE_TEXTURE
      * @member NODE_TEXTURE_TYPE_TEXTURE_SET
      */
-    enum NodeTextureType
```

Comment:
this is cpp 11 feature, I think it's ok to use it?

Reply:
I think so too, yes.